src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuProcessPdpCancelPaidServiceImpl.java

```diff
@@ -4,77 +4,60 @@
 import java.util.Iterator;
 import java.util.List;
 
-import org.kuali.kfs.pdp.PdpConstants;
 import org.kuali.kfs.pdp.batch.service.impl.ProcessPdpCancelPaidServiceImpl;
 import org.kuali.kfs.pdp.businessobject.ExtractionUnit;
 import org.kuali.kfs.pdp.businessobject.PaymentDetail;
-import org.kuali.kfs.sys.batch.service.PaymentSourceToExtractService;
-import org.kuali.kfs.sys.document.PaymentSource;
-import org.kuali.rice.kew.api.exception.WorkflowException;
-import org.kuali.kfs.krad.util.ObjectUtils;
+import org.springframework.transaction.annotation.Transactional;
 
-import edu.cornell.kfs.integration.purap.CuPurchasingAccountsPayableModuleService;
-import edu.cornell.kfs.pdp.businessobject.PaymentDetailExtendedAttribute;
+import edu.cornell.kfs.pdp.batch.service.ProcessPdpCancelPaidHelperService;
 
+@Transactional
```

Comment:
I am not sure why we need this annotation at the service level?

Reply:
I was trying to be consistent with the ProcessPdpCancelPaidServiceImpl superclass, which also has this annotation at the class/service level.

Reply:
Due to the refactoring changes I'm currently making, I will remove this class-level annotation for now and see if things still work.